vllm/model_executor/weight_utils.py

```diff
@@ -134,7 +136,8 @@ def prepare_hf_model_weights(
     revision: Optional[str] = None,
 ) -> Tuple[str, List[str], bool]:
     # Download model weights from huggingface.
-    is_local = os.path.isdir(model_name_or_path)
+    is_local = os.path.isdir(
```

Comment:
Escape with `\` after `)`
```
... isdir(model_name_or_path) \
       and load_format != "tensorizer"
```

Reply:
Alternatively, if you encapsulate the expression in parens you don't need the line continuation `\`, i.e. 

```python
is_local = (os.path.isdir(model_name_or-path)
    and load_format != "tensorizer")
```

also this is more of a stylistic thing so take it or leave it, but whenever I have a multi-expression conditional like this I generally try to encapsulate each subexpression in parens to mitigate order of operations ambiguity. So not a required change, but if you wanna be extra explicit you could do:

```python
is_local = (os.path.isdir(model_name_or-path)
    and (load_format != "tensorizer"))
```

Reply:
I like both suggestions! I'll implement this